dune_client/file.py

```diff
@@ -82,6 +86,18 @@ def write(self, out_file: TextIO, data: list[DuneRecord]) -> None:
             raise ValueError(f"Unrecognized FileType {self} for {out_file.name}")
 
 
+# def skip_empty(func):
+#     """Decorator used on write methods"""
+#
+#     def write_wrapper(self, data, name, ftype):
+#         if len(data) == 0:
+#             logger.info(f"Nothing to write to {name}... skipping")
+#             return
+#         func(self, data, name, ftype)
+#
+#     return write_wrapper
+
+
```

Comment:
Forgotten?

Reply:
Right. that is the content of this issue and another comment below:

https://github.com/cowprotocol/dune-client/issues/45